marvinIsSacul/url-short
# Feature/url short

## Description

This PR implements a new feature of the URL shortening service


## Testing Instructions

1. Clone the repository.
2. Checkout this branch (`feature/url-short`).
3. Start the server.
4. Use a tool like Postman to send a GET request to `/l/:hash with a valid short URL identifier.
5. Verify that the response includes the correct data for the URL.

## Checklist

- [x] Implemented the URL shortening service.
- [x] Added relevant documentation.
- [x] Added Postman collection
